ctags.lua

```diff
@@ -249,6 +258,13 @@ vis:command_register("pop", function(argv, force, win, selection, range)
 	pop_pos()
 end)
 
+vis:option_register("tags", "string", function(value)
+	tags = {}
+	for str in value:gmatch('([^%s]+)') do
+		table.insert(tags, str)
+	end
+end, 'paths to search for tags')
```

Comment:
```suggestion
end, 'Paths to search for tags (separated by spaces)')
```

Reply:
Resolved in 0837d78c3bd0fe354eba815e8a142f97264867e9